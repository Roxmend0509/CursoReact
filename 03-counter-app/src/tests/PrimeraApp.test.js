import PrimeraApp from "../PrimeraApp";
import React from 'react';

import '@testing-library/jest-dom';

import {shallow} from 'enzyme';


describe('Pruebas en <PrimeraApp />',()=>{
//     test('Debe de mostrar el mensaje "Hola soy nene"',()=>{
//         const saludo = "Hola soy nene";

//         const {getByText} = render(<PrimeraApp saludo={ saludo }/>);

//         expect(getByText(saludo)).toBeInTheDocument();
//     })

    test('debe mostrar <Primera /> correctamente', () => { 
        const saludo = "Hola soy nene";
        const wrapper = shallow(<PrimeraApp saludo={ saludo }/>);
        expect(wrapper).toMatchSnapshot();
    })
    })