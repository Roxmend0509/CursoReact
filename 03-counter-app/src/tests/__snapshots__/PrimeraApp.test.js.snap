// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> debe mostrar <Primera /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola soy nene
    !!!
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
